FileShareClient/FileShareApiClient.cs

```diff
@@ -99,6 +100,40 @@ public async Task<BatchSearchResponse> Search(string searchQuery, int? pageSize
             }
         }
 
+        public async Task<IResult<BatchSearchResponse>> Search(string searchQuery, CancellationToken cancellationToken, int? pageSize = null, int? start = null)
```

Comment:
Please change the optional parameters to mandatory
```suggestion
        public async Task<IResult<BatchSearchResponse>> Search(string searchQuery, CancellationToken cancellationToken, int? pageSize, int? start)
```

Reply:
resolved